### PR TITLE
add Blind SSRF with proxy param or url param

### PR DESCRIPTION
**Pull request by Jumbo-WJB**

### CODE REVIEW

File: other/Blind-SSRF-By-Collaborator.bcheck [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    author: "Jumbo@chinabaiker.com"
+
+
+define:
+    proxy_ssrf = `http://{generate_collaborator_address()}/proxy`
+    url_ssrf = `https://{generate_collaborator_address()}/url`
+given request then
+    send request:
+        appending queries:
+				  `proxy={proxy_ssrf}`,
```
Comment (PortSwiggerWiener) [inline other/Blind-SSRF-By-Collaborator.bcheck:14]: can you please fix the indentation here?

File: other/Blind-SSRF-By-Collaborator.bcheck [hunk 2]
```diff
@@ -0,0 +1,23 @@
+metadata:
+    language: v1-beta
+    name: "Blind SSRF By Collaborator"
+    description: "Blind SSRF with proxy param or url param"
+    author: "Jumbo@chinabaiker.com"
+
+
```
Comment (PortSwiggerWiener) [inline other/Blind-SSRF-By-Collaborator.bcheck:7]: Please can you remove the extra line here?

File: other/Blind-SSRF-By-Collaborator.bcheck [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+define:
+    proxy_ssrf = `http://{generate_collaborator_address()}/proxy`
+    url_ssrf = `https://{generate_collaborator_address()}/url`
+given request then
```
Comment (PortSwiggerWiener) [inline other/Blind-SSRF-By-Collaborator.bcheck:11]: Can you add an extra line here?

**Review by PortSwiggerWiener — COMMENTED** [review]
Looks good. Please can you address the cosmetic issues and I'll approve. 👍

**Comment by Jumbo-WJB** [comment]
yes i have fixed:4e1ba7df6ef5ab0f0f7d3531e52c5d2ae0350327

**Review by PortSwiggerWiener — APPROVED** [review]
Thanks for the modifications. 👍

**Review by A-J-C — APPROVED** [review]

**Comment by michael-eaton-portswigger** [comment]
@Jumbo-WJB As a contributor to our GitHub repository, we would like to invite you to our closed Discord community.

It is a place where passionate Burp users, including people who directly work on building and developing Burp here at PortSwigger, can talk about the tooling and web security in general.

If you would like to join, please email us at [support@portswigger.net](mailto:support@portswigger.net) and we will send over an invite link.

Thank you!

**Pull request merged**